addons/binding/org.openhab.binding.dscalarm/src/main/java/org/openhab/binding/dscalarm/handler/PartitionThingHandler.java

```diff
@@ -155,22 +155,22 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                     int partitionNumber = getPartitionNumber();
                     if (command.toString().equals("0")) {
                         dscAlarmBridgeHandler.sendCommand(DSCAlarmCode.PartitionDisarmControl, String.valueOf(partitionNumber));
-                        updateProperties(channelUID, 0, "Partition Disarmed");
+                        // updateProperties(channelUID, 0, "Partition Disarmed");
```

Comment:
why are all those commented out (if not needed, better remove the lines)?
And btw, this looks a lot as if you are misusing Thing properties for some local storage - this is not really the idea of them...
What exactly are you using them for? Is there another way to achieve this?